eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -49,20 +51,42 @@ class SqliteAuditDb(sqlite: Connection) extends AuditDb with Logging {
       statement.executeUpdate("CREATE INDEX IF NOT EXISTS channel_errors_timestamp_idx ON channel_errors(timestamp)")
     }
 
+    def migration34(statement: Statement) = {
+      statement.executeUpdate("DROP index sent_timestamp_idx")
+      statement.executeUpdate("ALTER TABLE sent RENAME TO _sent_old")
+      statement.executeUpdate("CREATE TABLE sent (id BLOB NOT NULL, amount_msat INTEGER NOT NULL, fees_msat INTEGER NOT NULL, payment_hash BLOB NOT NULL, payment_preimage BLOB NOT NULL, timestamp INTEGER NOT NULL)")
```

Comment:
I would keep the field `to_channel_id` and only input sub-payments in case of AMP. We can easily group subpayments using the `id` field.

Reply:
I'll re-work that part, that makes sense.

Reply:
Actually I'm not sure it makes sense. I think it's better that AuditDb doesn't know about the internal details that a payment was AMP (this is an implementation detail that you can ignore when you only want to look at things from a high-level view, especially since either all sub-payments complete or none of them complete).

The sub-payment details (with the whole route, hence the channelId) are already in the PaymentsDb if you want to have a more detailed look at how the payment was split and what routes it took. Let's discuss that offline.

Reply:
As discussed offline, I'll investigate this further ;)